stdlib/src/builtin/uint.mojo

```diff
@@ -158,6 +159,17 @@ struct UInt(IntLike):
         # TODO(MOCO-636): switch to DType.index
         return _hash_simd(Scalar[DType.uint64](self))
 
+    fn __hash__[H: _Hasher](self, inout hasher: H):
+        """Updates hasher with this uint value.
+
+        Parameters:
+            H: The hasher type.
+
+        Args:
+            hasher: The hasher instance.
+        """
+        hasher._update_with_simd(Scalar[DType.uint64](self))
```

Comment:
Ditto.

Reply:
I fixed this up internally before landing it.